docs/source/chatops/msteams_is_a_diva.rst

```diff
@@ -0,0 +1,234 @@
+Using Microsoft Teams (with BotFramework)
```

Comment:
I strongly recommend not using a page name like that. At some point that comment might not apply, and changing page names changes links, which means you'll need to add redirect pages or Google will point to the old content forever

Reply:
Fixed.